services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -195,6 +195,35 @@ const indexBlock = async job => {
 					}, dbTrx);
 				}
 			}
+
+			// TODO: Verify and enable it once pos:validatorStaked schema is exposed from SDK
+			// const tokenIDLockedAmountChangeMap = {};
+			// events.forEach(event => {
+			// 	const { data: eventData } = event;
+			// 	// Initialize map entry with BigInt
+			// 	if (['locked', 'unlocked'].includes(event.name) &&
+			//		!(eventData.tokenID in tokenIDLockedAmountChangeMap[eventData.tokenID])) {
+			// 		tokenIDLockedAmountChangeMap[eventData.tokenID] = BigInt(0);
+			// 	}
+
+			// 	if (event.name === 'locked') {
+			// 		tokenIDLockedAmountChangeMap[eventData.tokenID] += eventData.amount;
+			// 	} else if (event.name === 'unlocked') {
+			// 		tokenIDLockedAmountChangeMap[eventData.tokenID] -= eventData.amount;
+			// 	}
+			// });
```

Comment:
Please consider defining a const `EVENT` map like shown below and replacing the hard-coded constants:

```
const EVENT = Object.freeze({
 	LOCKED: 'locked',
 	UNLOCKED: 'unlocked',
});
```

```suggestion
			// 	if ([EVENT.LOCKED, EVENT.UNLOCKED].includes(event.name) &&
			//		!(eventData.tokenID in tokenIDLockedAmountChangeMap[eventData.tokenID])) {
			// 		tokenIDLockedAmountChangeMap[eventData.tokenID] = BigInt(0);
			// 	}

			// 	if (event.name === EVENT.LOCKED) {
			// 		tokenIDLockedAmountChangeMap[eventData.tokenID] += eventData.amount;
			// 	} else if (event.name === EVENT.UNLOCKED) {
			// 		tokenIDLockedAmountChangeMap[eventData.tokenID] -= eventData.amount;
			// 	}
			// });
```

Reply:
Applied.